Remove consensus behaviour from new products (#16662)
PostHog/posthog.com: contents/handbook/which-products.md

  - For example, a CRM. We'd be more excited about building a customer support tool, as support often is a task that involves engineering.
- Products that lean into 2020 style products, not 2030 style products
  - Think lots of UX for humans to use

## How new products get built

Sometimes the Blitzscale team will decide a new product needs to be built. They'll find someone internally to run it, ideally someone who's been at PostHog for at least 6 months (we tried getting new people to ship new products, but they often struggled to ship quickly).

Other times you might have an idea for a great product we should build. In that case, use the <PrivateLink url="https://github.com/PostHog/requests-for-comments-internal/blob/main/_TEMPLATES/request-for-comments-new-product.md">New Product RFC template</PrivateLink>. You might choose to hack together a prototype of the product to demo and show off, which you should do! Blitzscale only needs to get involved if you want to start working on this product full time. At that point, we are choosing whether to invest a pretty serious amount of money into launching it, so we want to get that right.

> The best products are often ones that not everyone thinks is a good idea before launch. For example, Tim was against building Session Replay, and it's probably the thing PostHog is most known for today. To make sure we take big bets, _any_ blitzscale team member can OK a new product being built, even if other blitzscale members disagree. That blitzscale person will be the one who the new team reports to. This way we avoid consensus stopping us from making big bets.

Reviewer: `s/thinks is a good/thinks are a good/`

> The best products are often ones that not everyone thinks is a good idea before launch. For example, Tim was against building Session Replay, and it's probably the thing PostHog is most known for today. To make sure we take big bets, _any_ blitzscale team member can OK a new product being built, even if other blitzscale members disagree. That blitzscale person will be the one who the new team reports to. This way we avoid consensus stopping us from making big bets.

Reviewer 2: ```suggestion
The best products are often ones that not everyone thinks is a good idea before launch. For example, Tim was against building Session Replay, and it's one of our most popular products. To make sure we take big bets, _any_ Blitzscale team member can OK a new product being built, even if other Blitzscale members disagree. That Blitzscale person will be the one who the new team reports to. This way we avoid consensus stopping us from making big bets.
```

Reviewer 2: (judging by #brand-mentions, people still overwhelmingly talk about PostHog for analytics above other stuff)
